tests/Alcaeus/MongoDbAdapter/MongoClientTest.php

```diff
@@ -82,7 +82,8 @@ public function testWriteConcern()
 
     public function testListDBs()
     {
-        $this->getCollection()->insert(['foo' => 'bar']);
+        $document = ['foo' => 'bar'];
+        $this->getCollection()->insert($document);
```

Comment:
Curious, but would `$this->getCollection()->insert($_ = ['foo' => 'bar']);` work as shorthand in the case where you don't care about the return value?